### PR TITLE
Add support for containerized badfish

### LINKED ISSUE #452: Use badfish container instead of badfish.py script
The official recommendation on the badfish repository is to use the badfish container[1]. They also do not plan to support the badfish.py script in the long term[2].
For this reason, Jetpack should switch to using the badfish container instead of the badfish.py script.

[1] https://github.com/redhat-performance/badfish#badfish-container
[2] https://github.com/redhat-performance/JetSki/issues/186#issuecomment-982666646

### PR DESCRIPTION
**Pull request by rajeshP524**
This PR resolves https://github.com/redhat-performance/jetpack/issues/452

### CODE REVIEW

**Comment by sanjaychari** [comment]
Update the commit messge to mention that this PR resolves https://github.com/redhat-performance/jetpack/issues/452.

**Comment by rajeshP524** [comment]
> Update the commit messge to mention that this PR resolves #452.

done.

File: common.yml [hunk 1]
```diff
@@ -65,6 +65,10 @@
         with_sequence: 0-{{ (oc_instackenv_content.nodes|length - 1) }}
         register: host_list
 
+      - name: pull badfish image
```
Comment (asyedham) [inline common.yml]: can you move this task to bootstrap.yml  playbook as common.yml  playbook contains only tasks that are needed by all the tags

**Review by asyedham — CHANGES_REQUESTED** [review]

**Review by asyedham — APPROVED** [review]
LGTM!

**Pull request merged**